Src/Module/LuaScript/LuaInterpreter/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ endif()
 if(ORBITER_BUILD_XRSOUND)
   add_dependencies(LuaInterpreter	XRSound_lib	XRSound_assets)
   target_link_libraries(LuaInterpreter XRSound_lib)
+  target_compile_definitions(LuaInterpreter PUBLIC XRSOUND)
```

Comment:
If this comes from XRSound, it needs to be a public definition on that target

Reply:
It doesn't come from XRSound. It's only passed to lua_xrsound.cpp to disable linking to XRSound if module doesn't exists to fix some link errors.